main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
Oh, and instead of hard-coding :2015, let's use the constant found in the config package: https://github.com/mholt/caddy/blob/4e1717db4c9b44c9255e5c58fce7d3f49af010a1/config/config.go#L18 (I forgot to tell you about this, sorry)